### DISCUSSION TITLE
New OBS Update managed to make the obs-multi-rtmp not working.

### DISCUSSION THREAD

**Discussion by KingMarth64** [sorayuki/obs-multi-rtmp#193] [Q&A]
Ok, I did managed to get a brand new update window popped up when I booted up OBS Studios and the new OBS studios managed to made new UI interface today. But, there was one major problem that I just happened to run into. The new OBS update managed to make the obs-multi-rtmp not working anymore and I did tried using the updated release of obs-multi-rtmp through the Tutorial on YouTube, it doesn't seem to work anymore on the new OBS update. So that means that I can't multistream on Twitch nor Facebook today while I streaming on YouTube.

Does anyone know how to get the obs-multi-rtmp from the new OBS update so far or we probably getting screwed till the creator of obs-multi-rtmp fix this issue through getting it working on the new OBS update?

<img width="1277" alt="Screenshot 2022-09-06 190647" src="https://user-images.githubusercontent.com/112996675/188774134-b2e206c5-f4a8-4047-bdbb-d4254c059342.png">

**Comment by Egorckaaa** [comment]
I have the same question

**Comment by sorayuki** [comment]
try unstall the old one.
Delete C:\ProgramData\obs-studio\plugins\obs-multi-rtmp if it exists.
Delete (where your obs installed)\obs-plugins\64bit\obs-multi-rtmp.dll and (where your obs installed)\data\obs-plugins if it exists

Then install the new one
________________________________

**Reply by Dputz** [reply]
We are having the same issue, Windows machine.  And there is no obs-multi-rtmp.dll found in either folder.  When doing a search that file name is not found.

**Reply by dgamefather** [reply]
Seeing this same issue with a friend's machine

**Comment by sorayuki** [comment]
Update to OBS 28.0.1 may helps
________________________________

**Reply by Dputz** [reply]
I am updated to 28.0.1 (64 bit), I updated this morning.  When I go into Computer A "OBS-Multi-rtmp" folder but in Computer B, I just installed the new version it is missing the folder.   

I am in 
C:\Program Files (x86)\obs-studio\data\obs-plugins and  
C:\Program Files (x86)\obs-studio\obs-plugins\64bit

**Reply by Dputz** [reply]
I was able to copy in a few items and it seems to be working. I also updated my RTMP keys for FB

**Comment by djquartz** [comment]
I just installed the latest plug-in and all my previous connections are gone. Can I get them back?

**Comment by bettungdl** [comment]
some one has news about this?

**Comment by 997879131** [comment]
Please follow the new OBS-Studio-30.0.2 complete installation program x64.exe multiple streaming plugins. Thank you
